docs/tutorials/projects/bedtime-songs-bot.md

```diff
@@ -0,0 +1,567 @@
+{
+ "components": [
+   {
+     "model": "webcam",
+     "attributes": {},
+     "depends_on": [],
+     "service_config": [
+       {
+         "attributes": {
+           "capture_methods": [
+             {
+               "additional_params": {
+                 "mime_type": "image/jpeg"
+               },
+               "capture_frequency_hz": .333,
+               "method": "ReadImage"
+             }
+           ]
+         },
+         "type": "data_manager"
+       }
+     ],
+     "name": "cam",
+     "type": "camera"
+   }
+ ],
+ "services": [
+   {
+     "attributes": {
+       "sync_interval_mins": 0.1,
+       "capture_dir": "",
+       "tags": []
+     },
+     "name": "Data-Management-Service",
+     "type": "data_manager"
+   }
+ ]
+}
+```
+
+Make sure that you have added a `service_config` to the JSON configuration of your webcam with type `data_manager`, as well as a new service with type `data_manager`, as shown above.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Capture data
+
+Your webcam is now configured to automatically capture images when you are connected to your robot live on [the Viam app](https://app.viam.com).
+At this point, grab the toys or any objects you want the robot to be able to differentiate between.
+
+Tess's kids like playing with puzzle pieces, which come in different shape and color combinations.
+They decided to filter between these puzzle pieces by tagging by shape, but you can filter your objects as you choose.
+
+Hold up the toys to the camera while photos are being taken.
+Try to capture images from different angles and backgrounds.
+Try to get at least 50 images that fit your criteria for each tag.
+
+You set the rate of capture in your webcam's service configuration attribute `capture_frequency_hz`.
+If you set this to `.333`, the data management service will capture 1 image roughly every 3 seconds as you hold up your toys to the camera.
+
+Go to the [**DATA tab**](https://app.viam.com/data/view?view=images) in the Viam app to see the images captured by your webcam.
+
+When you've captured enough images to tag, navigate back to the **Config** tab.
+Scroll to the card with the name of your webcam and click the power switch next to the **Data Capture Configuration** to **off** to [disable data capture](/services/data/configure-data-capture/#configure-data-capture-for-individual-components).
+
+Now, use these pictures to train your machine learning model.
+
+### Tag data
+
+Head over to the [**DATA** page](https://app.viam.com/data/view) and select an image captured from your robot.
+After selecting the image, you will see all of the data that is associated with that image.
+
+[Add tags](/manage/data/label/#image-tags) for each of the puzzle pieces.
+Type in your desired tag in the **Tags** section and save the tag.
+Since Tess wanted to classify their toys by shape, they used “octagon,” “circle,” “triangle,” “oval,” “rectangle,” “pentagon,” “diamond,” and “square”.
```

Comment:
```suggestion
Since Tess wanted to classify their toys by shape, they used `“octagon”`, `“circle”`, `“triangle”`, `“oval"`, `“rectangle”`, `“pentagon”`, `“diamond”`, and `“square”`.
```